src/libpgmoneta/utils.c

```diff
@@ -3003,6 +3004,128 @@ pgmoneta_is_file_archive(char* file_path)
    return false;
 }
 
+/* Parser for pgmoneta-cli amd pgmoneta-admin commands */
+bool
+parse_command(int argc,
+              char** argv,
+              int offset,
+              struct pgmoneta_parsed_command* parsed,
+              const struct pgmoneta_command command_table[],
+              size_t command_count)
+{
+#define EMPTY_STR(_s) (_s[0] == 0)
```

Comment:
Hmm, isn't

```
if (s != NULL && strlen(s) > 0)
```

better ?

Reply:
I think it makes the code a bit cleaner if we use macros. Subcommand is usually empty and we do the check frequently.